mgmt/Rollback.cc

```diff
@@ -34,12 +34,6 @@
 #include "FileManager.h"
```

Comment:
Why do we still have this file?

Reply:
You mean Rollback? Turns out the rollback object was both for rollbacks as well as change management. Change management is still there. I can rename this file if preferable.